.github/workflows/build-docker-images.yaml

```
@@ -0,0 +1,74 @@
+name: Build Docker Images
+
+description: Build Docker images for the Riven app and optionally push them to the registry.
```

actionlint (1.7.12)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)